### PR TITLE
Added: Check server status method added for the appwrite server before going to the home screen.

### LINKED ISSUE #211: Check server status before going to home screen
### 🚀 The feature

We will have to check if our appwrite server instance is up and running.

### PR DESCRIPTION
**Pull request by xonx4l**
issue-:#211

## Description

Added: Check server status method added for the appwrite server before going to the home screen.

Fixes #211

## Type of change

- [x ] New feature (non-breaking change which adds functionality)

## Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by chandansgowda — CHANGES_REQUESTED** [review]
We will have to call this function and if it returns false, have to navigate the user to a screen where we tell the user that our server is down for some reason.

**Comment by xonx4l** [comment]
@chandansgowda review as if I am going correct .

File: lib/services/appwrite_service.dart [hunk 1]
```diff
@@ -36,3 +36,31 @@ class AppwriteService {
...
+
+// Function to check server status and navigate accordingly
+  static Future<void> checkServerAndNavigate(BuildContext context) async {
+    bool isServerUp = await checkServerStatus();
+
+    if (isServerUp) {
+      // Server is up, you can proceed with your normal flow
+    } else {
+      // Server is down, navigate to a screen informing the user
+      Navigator.push(
```
Comment (chandansgowda) [inline lib/services/appwrite_service.dart:59]: We are using GetX for state management.

**Review by chandansgowda — CHANGES_REQUESTED** [review]
Th function you have written has to be called while the splash screen initialises.

**Comment by Aarush-Acharya** [comment]
Hey @xonx4l  please do the requested changes for us to merge the PR

**Comment by Aarush-Acharya** [comment]
Hey @xonx4l  i believe the change required is very minor can be dealt with later on, so merging this PR

**Pull request merged**